magento/pwa-studio
# Merge main back to develop

# ❗ DO NOT SQUASH MERGE THIS PR ❗ 



## Description
Merge 12.2 main back to develop

## Related Issue







Closes #ISSUE_NUMBER.

## Acceptance





### Verification Stakeholders



### Specification




## Verification Steps






#### Test scenario(s) for direct fix/feature





1. Run cypress smoke test
2. Verify changes from `main` are in develop. 

## Screenshots / Screen Captures (if appropriate)

## Breaking Changes (if any)





## Checklist



-   I have added tests to cover my changes, if necessary.
-   I have added translations for new strings, if necessary.
-   I have updated the documentation accordingly, if necessary.
